### PR TITLE
OPSEXP-1661:  Author new github action on ansible repository to avoid duplications in workflows

### PR DESCRIPTION
**Pull request by piraartur**
OPSEXP-1661

### CODE REVIEW

File: .github/actions/pipenv/action.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+description: 'Install python packages and create pipenv'
+
+runs:
+  using: "composite"
+  steps: 
+  - name: Install pipenv and Pipfile
+    shell: bash
+    run: |
+      pip install --user pipenv
+      pipenv install --deploy --dev
```
Comment (gionn) [inline .github/actions/pipenv/action.yml:11]: split in two different step so the output will be more readable

File: .github/actions/pipenv/action.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: install_python_packages
```
Comment (gionn) [inline .github/actions/pipenv/action.yml:1]:
```suggestion
name: pipenv
```

usually the name of the upper folder or a shorter description

File: .github/actions/pipenv/action.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+description: 'Install python packages and create pipenv'
```
Comment (gionn) [inline .github/actions/pipenv/action.yml:2]:
```suggestion
description: 'Install python packages with pipenv'
```

**Review by gionn — CHANGES_REQUESTED** [review]

**Review by kklassa — APPROVED** [review]
looks fine in my opinion

**Review by gionn — APPROVED** [review]

**Pull request merged**